conda/meta.yaml

```diff
@@ -1,5 +1,5 @@
 {% set name = "pyboolnet" %}
-{% set version = "2.3.0" %}
```

Comment:
so every time I make a release I have to adjust this number?

Reply:
Right, now we can also pull it fro setup.py, I'll change this.
Some of the dependencies can be managed automatically, but not all of them

Reply:
Ok great!